### DISCUSSION TITLE
Support for XDG activation in qtile.

### DISCUSSION THREAD

**Discussion by ftpd** [qtile/qtile#4161] [Ideas]
After switching to Wayland, I've switched to [foot](https://codeberg.org/dnkl/foot) terminal emulator. To get notifications/urgency for it 'visible' in my Group widget (I would like to have it on my `urgent_*` colors that I can set in the widget) and [I was told](https://codeberg.org/dnkl/foot/issues/1282) the compositor has to enable [XDG activation support](https://wayland.app/protocols/xdg-activation-v1). Can we have this in qtile, please?

**Comment by m-col** [comment]
For sure! We definitely need this. It's on my to do list ([here](https://github.com/orgs/qtile/projects/2/views/2)).
